src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -75,7 +75,7 @@ static bool KeyExists(FabricIndex fabricIndex, GroupId groupId)
 
 static Status GroupAdd(FabricIndex fabricIndex, EndpointId endpointId, GroupId groupId, const CharSpan & groupName)
 {
-    VerifyOrReturnError(IsFabricGroupId(groupId), Status::ConstraintError);
+    VerifyOrReturnError(IsApplicationGroupId(groupId), Status::ConstraintError);
```

Comment:
My understanding is that the spec purposefully allowed all group id values >= 1 here.  At least that's what https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4081#issuecomment-1018000764 seems to say, and what the appclusters spec seems to say, no?

As in, this test should probably be IsOperationalGroupId, or just >= 1, depending on intent...

Reply:
Changed this check to `IsOperationalGroupId`

Reply:
Since the appclusters spec says we should return CONSTRAINT_ERROR when the command fields are not within constraints so I changed this check to IsValidGroupId at last.